Dynatrace/dynatrace-operator
# Implements E2E test for using CodeModules

# Description

Implements a test to check whether the CSI driver can correctly download a codemodules image and reuse its content for different tenants and dynakubes.

## How can this be tested?

* Create a cluster
* Run the test
* Observe what it does and make sure it does the right things

## Checklist
